File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    result = [];
    for i in range(l, r+1):
        temp = list(str(i));
        isFive = list(map(lambda el: 'True' if el == '0' or el == '5' else 'False', temp));
        if 'False' not in isFive: result.append(i)
        
    return [-1] if len(result) == 0 else result

File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.js
function solution(l, r) {
    const result = []
    for(let i = l; i <= r; i++){
        let temp = String(i).split('')
        let isFive = temp.map(el => el === '0' || el === '5' ? true : false)
        if(isFive.includes(false)) continue;
        else result.push(i)
    }

    return result.length === 0 ? [-1] : result
}